<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1779</width>
    <height>780</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>769</width>
    <height>455</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>H-Tech Monitor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/main/monitor_icon.png</normaloff>:/main/monitor_icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-color: #121212;
}

QStatusBar{
	color: white;
}

QWidget[state=&quot;scroll&quot;]{
	background-color: #121212;
}

QPushButton:enabled{
	background-color: #121212;
	color: white;
}

QPushButton:enabled{
	/*background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #121212, stop: 1 #121212);*/
	/*background-color: #121212;*/
	/*color: white;*/
	/*font: 10pt &quot;Verdana&quot;;*/
}

QGroupBox{	
	border: none;
}

QTableWidget::item:hover{
	color: white;
}

 QScrollBar:vertical {
     border-radius: 100px;
     background: #595959;
     width: 5px;
     margin: 22px 0 22px 0;
 }
 QScrollBar::handle:vertical {
     background: #cdcdcd;
     min-height: 20px;
 }
 QScrollBar::add-line:vertical {
     /*border: 2px solid grey;*/
     /*background: #32CC99;*/
     /*height: 20px;*/
    /* subcontrol-position: bottom;*/
     /*subcontrol-origin: margin;*/
	width: 0;
 }

 QScrollBar::sub-line:vertical {
   /*  border: 2px solid grey;*/
     /*background: #32CC99;*/
     /*height: 20px;*/
    /* subcontrol-position: top;*/
    /* subcontrol-origin: margin;*/
	width: 0;
 }
 QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
     border: 2px solid grey;
     width: 3px;
     height: 3px;
     background: white;
 }

 QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
     background: none;
 }

QTableWidget{
	selection-color: #2980b9;
	selection-background-color: #121212;
	background-color: #121212;
	color: #7f8c8d;
}

QComboBox{
	color: #7f8c8d;
	background: #121212;	
	font: 75 12pt &quot;Verdana&quot;;
}

QComboBox::focus{
	color: white;
	selection-background-color: #121212;
}

Line{
	border: 2px solid #3A3A3A;
}</string>
  </property>
  <property name="iconSize">
   <size>
    <width>50</width>
    <height>50</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>256</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QComboBox" name="selectorSubsistema">
           <property name="font">
            <font>
             <family>Verdana</family>
             <pointsize>12</pointsize>
             <weight>9</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QListWidget::item:hover{	
	background-color: rgb(170, 0, 0);
}</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="botonLogin">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <underline>true</underline>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Iniciando sesión...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6"/>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="botonSeleccionarUno">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Seleccion singular</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	background-color: 7f8c8d;
}
QPushButton:checked{
	background-color: #3498db;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/buttons/icons/show_one.png</normaloff>:/buttons/icons/show_one.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="botonSeleccionarVarios">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Selección múltiple</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	background-color: 7f8c8d;
}
QPushButton:checked{
	background-color: #3498db;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/buttons/icons/show_multiple.png</normaloff>:/buttons/icons/show_multiple.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="botonOcultarTodos">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Ocultar todos</string>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: 7f8c8d;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/buttons/icons/show_none.png</normaloff>:/buttons/icons/show_none.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="botonMostrarTodos">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Mostrar todos</string>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: 7f8c8d;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/buttons/icons/show_all.png</normaloff>:/buttons/icons/show_all.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tablaBombas">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">/*selection-color: #FFFFFF;
selection-background-color: #121212;
background-color: #121212;
color: #7f8c8d;*/</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>false</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Nombre</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Ubicación</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1488</width>
           <height>671</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2"/>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <family>Verdana</family>
    </font>
   </property>
  </widget>
  <action name="deshacer">
   <property name="text">
    <string>Deshacer(Ctrl+Z)</string>
   </property>
  </action>
  <action name="rehacer">
   <property name="text">
    <string>Rehacer(Ctrl+Y)</string>
   </property>
  </action>
  <action name="abrir">
   <property name="text">
    <string>Iniciar sesión...</string>
   </property>
  </action>
  <action name="verSensores">
   <property name="text">
    <string>Ver sensores</string>
   </property>
  </action>
  <action name="editarSensores">
   <property name="text">
    <string>Editar sensores</string>
   </property>
  </action>
  <action name="about">
   <property name="text">
    <string>Acerca de H-Tech</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
